kubernetes-manifests/environment-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: environment-config
+data:
+  VERSION: "v0.1.0"
```

Comment:
Megan had mentioned it being beneficial to give each service it's own version string that we can manipulate during demos. Maybe we should have an APP_VERSION and SERVICE_VERSION and output both or something? Thoughts?

Reply:
Moved VERSION back to each service yaml.  We can decide if we need an APP_VERSION later.